bigbluebutton-html5/imports/ui/components/chat/notification/component.jsx

```diff
@@ -176,10 +185,19 @@ class ChatNotification extends Component {
       publicUserId,
       intl,
       disableNotify,
+      chatIsOpen,
+      currentChatID,
     } = this.props;
 
     const publicUnread = UnreadMessages.getUnreadMessages(publicUserId);
     const publicUnreadReduced = Service.reduceAndMapMessages(publicUnread);
+
+    if (disableNotify) return;
+    if (!Service.hasUnreadMessages(publicUserId)) return;
+    if (chatIsOpen && currentChatID === PUBLIC_KEY) return;
+
+    const setIntervalTime = (sender, time) => time > (this.state.publicNotified[sender.id] || 0);
```

Comment:
this name is very strange, it is checking if the message time is longer than the time of the last message being notified and being called `setIntervalTime`, it would be better to have a name like` isNotNotified` or `checkIfBeenNotified`

Reply:
Ok I will change it